src/sentry/static/sentry/app/views/releases/list/organizationReleases/index.jsx

```diff
@@ -24,6 +22,26 @@ import ReleaseListHeader from '../shared/releaseListHeader';
 import ReleaseLanding from '../shared/releaseLanding';
 import {getQuery} from '../shared/utils';
 
+class OrganizationReleasesContainer extends React.Component {
+  static propTypes = {
+    organization: SentryTypes.Organization,
+  };
+
+  render() {
+    const {organization} = this.props;
+    return (
+      <Feature
+        features={['organizations:sentry10']}
+        organization={organization}
+        renderDisabled={this.renderNoAccess}
```

Comment:
Where is this method defined?

Reply:
Thanks forgot to move these to the new class - updated